crypto/crypto.gradle

```diff
@@ -10,3 +10,7 @@ configure(project.tasks.withType(Test)) {
 		exclude '**/EncryptorsTests.class'
 	}
 }
+
+dependencies {
+    compile 'org.bouncycastle:bcprov-jdk15on:1.54'
```

Comment:
This should be optional. Also don't forget to regenerate the pom.xml using the following from this directory

../gradlew generatePom